Source/WebCore/style/values/primitives/StylePrimitiveNumericTypes+Conversions.h

```diff
@@ -188,7 +188,7 @@ template<auto R, typename... Rest> LengthPercentage<R> canonicalize(const CSS::L
 // MARK: - Conversion from "Style to "CSS"
 
 // Out of line to avoid inclusion of CSSCalcValue.h
-Ref<CSSCalcValue> makeCalc(const CalculationValue&, const RenderStyle&);
+Ref<CSSCalcValue> makeCalc(Ref<CalculationValue>, const RenderStyle&);
```

Comment:
How do you decide when to make an argument `Ref&&` and when to make it `Ref`? And should we write that into the coding style guidelines?